lib/import_govdelivery_csv.rb

```diff
@@ -98,7 +98,9 @@ def import_subscriptions
 
     records = CSV
       .foreach(subscriptions_csv_path, headers: true, encoding: "WINDOWS-1252")
-      .map do |row|
+      .each_with_index.map do |row, i|
```

Comment:
might want to pass in an argument of 1 to each_with_index (as offset) as I think it might all end up being off by one. Not really a biggie mind.